spark_log_parser/parsing_models/application_model_v2.py

```diff
@@ -105,7 +105,7 @@ def save_to_s3(self, saveDat, filepath, compress):
 DataType = TypeVar("DataType")
 
 
-class AbstractSparkApplicationDataLoader(abc.ABC, Generic[DataType], DataLoader):
+class AbstractSparkApplicationDataLoader(DataLoader, Generic[DataType], abc.ABC):
```

Comment:
In Python 3.11.1 this yields,
```
TypeError: Cannot create a consistent method resolution order (MRO) for bases object, Generic, DataLoader
```
It's typically better to put the less general types first.

Reply:
Ah, I was going to ask! Makes sense to me :+1: